packages/create-portal-app/helpers/replace.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
+const path = require('path')
+const { parserPath } = require('./copy')
+function _replacePackage(projectName) {
```

Comment:
I think space should be between ` const { parserPath } = require('./copy')` and  `function _replacePackage`

Reply:
sure.